<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1063</width>
    <height>956</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="minimumSize">
    <size>
     <width>1063</width>
     <height>795</height>
    </size>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>10</y>
      <width>371</width>
      <height>251</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_camera">
       <property name="font">
        <font>
         <family>Consolas</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>                pic</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label_display">
    <property name="geometry">
     <rect>
      <x>18</x>
      <y>18</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="sizeIncrement">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="baseSize">
     <size>
      <width>640</width>
      <height>480</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Microsoft JhengHei UI</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSplitter" name="splitter">
    <property name="geometry">
     <rect>
      <x>682</x>
      <y>271</y>
      <width>371</width>
      <height>631</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QPushButton" name="pushButton_convert">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>&amp;Convert</string>
     </property>
     <property name="shortcut">
      <string>Ctrl+C</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_image">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>open_image</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_camera">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>open_camera</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_yuyin">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>yuyin_test</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Capture">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Capture</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Save">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Quit">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Quit</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_input">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </widget>
   <widget class="QSplitter" name="splitter_2">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>510</y>
      <width>291</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>输入框</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit"/>
    <widget class="QLabel" name="label_5">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>拼音</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_4"/>
    <widget class="QLabel" name="label_6">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>双拼盲文</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_5"/>
   </widget>
   <widget class="QSplitter" name="splitter_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>510</y>
      <width>341</width>
      <height>391</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>汉字</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser">
     <property name="font">
      <font>
       <family>Microsoft YaHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>拼音</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_2">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>双拼盲文</string>
     </property>
    </widget>
    <widget class="QTextBrowser" name="textBrowser_3">
     <property name="font">
      <font>
       <family>Consolas</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1063</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
